.github/ISSUE_TEMPLATE/04_core_feature_request.yml

```diff
@@ -0,0 +1,62 @@
+name: Core Feature Request
+description: Create a feature request for Azure.Core
+title: "[Core] "
+labels: ["Azure.Core", "feature-request"]
+assignees: ["christothes","annelo-msft", "Josh-Love-msft"]
+
+body:
+  - type: dropdown
+    id: apichange
+    attributes:
+      label: API Change Needed
+      description: Is a public API change needed for this feature?
+      options:
+        - "Yes"
+        - "No"
+        - "Unsure"
+    validations:
+      required: true
+  - type: textarea
+    id: background
+    attributes:
+      label: Background and motivation
+      description: Please describe the purpose and value of the new feature and/or API here.
```

Comment:
Could we make this more Azure SDK-specific by asking specifically for:
- What service libraries drove the requirement, and what the scenario is
- What service libraries will need to adopt the change

Reply:
We could add it to this section https://github.com/Azure/azure-sdk-for-net/pull/27827/files#diff-4aecfafb7ad774892c793025c1a73e289e77e759c692d8477b2714f62b07fa1bR50

Reply:
Added some more text based on your suggestions